docs/dqx/docs/motivation.mdx

```diff
@@ -10,7 +10,7 @@ data quality issues and are primarily designed for complete datasets,
 making integration into streaming workloads difficult.
 They also lack the ability to quarantine invalid data and have compatibility issues with Databricks Runtime.
 
-This project introduces a simple Python validation framework for assessing data quality of PySpark DataFrames. 
+This project introduces a simple and powerful Python validation framework for assessing data quality of PySpark DataFrames.
```

Comment:
"simple and powerful" -> "simple but powerful"?

Reply:
corrected